packaging/installer/README.md

```diff
@@ -293,6 +293,28 @@ To apply the changes you made, you have to restart Netdata.
 
 ---
 
+### Binary Packages
+![](https://raw.githubusercontent.com/netdata/netdata/master/web/gui/images/packaging-beta-tag.svg?sanitize=true)
+
+We provide our own flavour of binary packages for the most common operating systems that comply with .RPM and .DEB packaging formats.
+We have currently released .RPM versions with version [1.16.0](https://github.com/netdata/netdata/releases/tag/v1.16.0). We are planning to release packages following the .DEB format on one of the following releases. Our current packaging infrastructure provider is [Package Cloud](https://packagecloud.io).
+Netdata is committed to support installation of our solution to all operating systems.
+This is a constant battle for Netdata, as we strive to automate and make things easier for our users.
+For the operating system support matrix, please visit our [distributions](../DISTRIBUTIONS.md) support page.
+
+We provide two separate repositories, one for our stable releases and one for our nightly releases.
+1. Stable releases
+
+   Our stable production releases are hosted in [netdata/netdata](https://packagecloud.io/netdata/netdata) repository of package cloud
+
+2. Nightly releases
```

Comment:
For both these, just combine the lines together, otherwise the formatting is hard to follow when built.

```
1. Stable releases: Our stable production releases are hosted in [netdata/netdata](https://packagecloud.io/netdata/netdata) repository of Package Cloud.

2. Nightly releases: Our latest releases are hosted in [netdata/netdata-edge](https://packagecloud.io/netdata/netdata-edge) repository of Package Cloud.
```